seatunnel-connectors-v2/connector-flink-table-store/src/main/java/org/apache/seatunnel/connectors/seatunnel/fts/config/FlinkTableStoreConfig.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.connectors.seatunnel.fts.config;
+
+import org.apache.seatunnel.api.configuration.Option;
+import org.apache.seatunnel.api.configuration.Options;
+
+import java.util.List;
+
+public class FlinkTableStoreConfig {
+
+    public static final Option<String> TABLE_PATH =
+            Options.key("table_path")
+                    .stringType()
+                    .noDefaultValue()
+                    .withDescription("The table path of flink table store");
+
+    public static final Option<List<String>> READ_COLUMNS =
+            Options.key("read_columns")
+                    .listType()
+                    .noDefaultValue()
+                    .withDescription("The read columns of the flink table store");
+
+    public static final Option<String> HDFS_SITE_PATH =
+            Options.key("hdfs_site_path")
```

Comment:
`hdfs_site_path` doesn't need to config because the path of hdfs site could configure via `HADOOP_CONF`.

Reply:
If the environment variable is not configured on the user's machine, hadoop configuration will not be loaded correctly, add the parameter can support user-defined the cluster want to connector. We can deal with use the following logic:

1. If user does configured this parameter, load the user-defined `hdfs-site.xml`
2. If user dose not configured this parameter, find the `HADOOP_CONF` env:
     - have finded, load the `HADOOP_CONF/hdfs-site.xml`
     - have not finded, do nothing, use the default empty configuration

What do you think about it?